plugin/trino-apache-ranger/pom.xml

```diff
@@ -198,6 +198,13 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.graalvm.sdk</groupId>
+            <artifactId>nativeimage</artifactId>
```

Comment:
cc: @kokosing

Reply:
It's an upper bound resolution. This dependency was already there

Reply:
CC: @mneethiraj